tmp/build/build.sh

```diff
@@ -14,5 +13,29 @@ mkdir -p ${BIN_DIR}
 PROJECT_NAME="istio-operator"
 REPO_PATH="github.com/maistra/istio-operator"
 BUILD_PATH="${REPO_PATH}/cmd/manager"
+
+: ${VERSION:=development}
```

Comment:
Should this be an actual version, even if for development, e.g. `${VERSION:-1.0.0-dev}

Reply:
It could be. RPM's will be overriding these variables. Default values are used only for developers while developing Operator. That said, I don't have a strong opinion. If we choose to do that, we should remember bumping it every new [even micro?] version.

Reply:
Yeah, development just seems really vague.  Let's leave it for now, dev can always override it on the build, `VERSION=1.x-dev .... ./tmp/build.sh`